IOMC/EventVertexGenerators/python/VtxSmearedParameters_cfi.py

```diff
@@ -624,6 +624,32 @@
     Z0 = cms.double(0.6707)
 )
 
+# From fit to 5 TeV pPb data
+Realistic5TeVPACollision2016VtxSmearingParameters = cms.PSet(
+    Phi = cms.double(0.0),
+    BetaStar = cms.double(1100.0),
+    Emittance = cms.double(6.75e-08),
+    Alpha = cms.double(0.0),
+    SigmaZ = cms.double(6.4891),
+    TimeOffset = cms.double(0.0),
+    X0 = cms.double(0.0889),
+    Y0 = cms.double(0.1820),
+    Z0 = cms.double(1.6066)    
+)
+
+# From fit to 8 TeV pPb data
+Realistic8TeVPACollision2016VtxSmearingParameters = cms.PSet(
+    Phi = cms.double(0.0),
+    BetaStar = cms.double(60.0),
+    Emittance = cms.double(6.75e-08),
+    Alpha = cms.double(0.0),
+    SigmaZ = cms.double(4.6914),
+    TimeOffset = cms.double(0.0),
+    X0 = cms.double(0.0836),
+    Y0 = cms.double(0.1837),
+    Z0 = cms.double(1.3577)    
+)
```

Comment:
@jrcastle, I was more concerned about the alignment used for the final fit in the MC samples. In this [request](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2775.html) you exaplain clearly the points:

1. An average beamspot was computed with Prompt RECO data using all
available runs for a given collision energy. The DCS JSON was also used
to filter lumis.

2. The centroid of this average beamspot was then translated to account
for the barycenter of the pixel tracker. See [1].

3. A new vertex smearing parameter set was included in CMSSW and a
"fake" payload was uploaded to the conditions database, each containing
the new, shifted beamspot. See [2].

4. The new vertex smearing parameters and fake payload were used to
simulate and reconstruct 100k MinBias pPb events.

5. A new fit of the beamspot was made to the MC data and uploaded to the
conditions database. This payload is the RECO BS we would like included
in a GT.

I would assume that the fake payload used in point 4. was closer to the vertex smearing than the final one in point 5.  Which conditions were used to reconstruct these 100k MinBias pPb events? I am asking because artificial of the BS sigmaZ is a known effect of exansion/shrinkage of the tracker alignment z-scale.